src/cdk-experimental/popover-edit/edit-event-dispatcher.ts

```diff
@@ -30,6 +31,12 @@ export class EditEventDispatcher {
   /** A subject that emits mouse move events for table rows. */
   readonly mouseMove = new Subject<Element|null>();
 
+  /** The EditRef for the currently active edit lens (if any). */
+  get editRef(): EditRef<any>|null {
```

Comment:
This could be in a follow-up change, but could we avoid using `any` here in favor of a generic?

Reply:
That would be tricky - the type is generic in the lens (refers to the shape of the form data) but different edits in the same table could have different lenses and this must refer to all of them.

It’d be fine to use unknown here but I don’t know what TS versions we have to support.

Reply:
We can make it unknown in a follow-up. As of Angular v8 we support 3.4 as the minimum